package.json

```diff
@@ -97,10 +97,13 @@
     "test-e2e-local": "node ./scripts/test-e2e-local.js",
     "test-e2e-local-clean": "node ./scripts/test-e2e-local-clean.js",
```

Comment:
headsup: we should migrate to TS also the other scripts in the folder like these two above 👆 is the plan to do it all at once or partially across multiple PRs?

Reply:
The plan for now is to migrate partially across multiple PRs, just for me to deliver some constant increment on this and not being fully invested in this JS -> TS migration. Is there any requirements on migrating all at once or one after another?

There is only one script I know for now which we won't be able to migrate, its `monorepo/find-and-publish-all-bumped-packages.js`, because it need to be executed without running `yarn`, so no `ts-node` will be present.

Reply:
ok sounds good - as long as we keep track of it all then I'm 👍 I think we can proceed as soon as we are sure that it's also ✅ from Meta side :)